src/benchmarks/micro/Serializers/Json_FromStream.cs

```diff
@@ -108,6 +108,27 @@ public T DataContractJsonSerializer_()
             return (T)dataContractJsonSerializer.ReadObject(memoryStream);
         }
 
+#if NETCOREAPP3_0 // API Available in .NET Core 3.0+
+        [GlobalSetup(Target = nameof(SystemTextJson_))]
+        public void SetupSystemTextJson_()
+        {
+            memoryStream.Position = 0;
+
+            using (var writer = new System.Text.Json.Utf8JsonWriter(memoryStream))
```

Comment:
nit: Consider moving the global setup next to the rest of the setup.

I realize that would result in more if/defs but maybe that's ok. I am also fine with it as is.

Reply:
I was 50/50 on this too - was just trying to minimize the #if noise as a default, but happy to do it either way you prefer!

Reply:
I'll let @adamsitnik, @billwert, @jorive chime in and decide which way they prefer.